README.md

```diff
@@ -0,0 +1,21 @@
+# Dex Lib
+
+## Description
+
+This is the tooling that allows you to connect new Dexes to `ParaSwap API`
+
+## Start
+
+- To prepare the project, run the commands:
+
+```bash
+git clone git@github.com:paraswap/paraswap-dex-lib.git
+yarn install
+```
+
+`Init` and `Test` new Dex:
+
+- `yarn init-integration your-dex-name` - specify the name of your Dex to initialize it
+- `yarn test-integration your-dex-name` - specify the name of the Dex to run tests for
+
```

Comment:
Awesome! can you add a sentence about where the new integration to be found after you init?

Reply:
Thank you very much. Added the sentence about the place of integration. Actually, I don't know why, but the selected code is a bit outdated. I changed slightly the structure of the README. But overall it is the same :)